src/commands/db/spec/db.spec.ts

```diff
@@ -20,15 +20,4 @@ describe(`sasjs db`, () => {
 
     await verifyStep('db')
   })
-
```

Comment:
Why you've decided to delete this test case?

Reply:
I have looked into this test case, it's not doing anything. There's no purpose of this test case.

Reply:
it checks if `buildDB()` throws a proper error message if `db` folder does not exis.

Reply:
I have re-written it correctly.